README.md

```diff
@@ -2,17 +2,77 @@
 
 ## About ##
 
-This is an attempt to build just another web-crawler, also called _Spider_. Its purpose is to get data from a website in a form of a list of objects. This data can be then used to download linked files and place them in a database or a folder structure.
+Onigumo je jeden z dalších web-crawlerů, také známých pod pojmem _Spider_.
```

Comment:
```suggestion
Onigumo je další web-crawler, také známých pod pojmem _Spider_.
```

Reply:
the term _spider_ is used in english sources see:

- https://www.makeuseof.com/what-is-a-web-crawlerspider-and-how-does-it-work/
- https://www.cloudflare.com/learning/bots/what-is-a-web-crawler/
- https://www.slant.co/topics/9435/~web-crawlers-spiders

Reply:
I note here because I wanna delete the sentence
```suggestion
Onigumo je jeden z dalších web-crawlerů.
```